communities/microgalaxy/lab/sections/3_microbiome.yml

```diff
@@ -1,9 +1,530 @@
+              Used in tools such as [<code>VirSorter</code>]({{ galaxy_base_url }}/root?tool_id=toolshed.g2.bx.psu.edu/repos/ufz/virsorter/virsorter)
+      
+      - id: general-sequence-databases
+        title: General Sequence Databases
+        content:
+          - title_md: <code>NCBI nt</code> - Nucleotide Sequence Database
+            button_link: "https://www.ncbi.nlm.nih.gov/nucleotide/"
+            button_tip: "Visit NCBI nt"
+            description_md: >
+              The NCBI nucleotide (nt) database is a comprehensive collection of nucleotide sequences from GenBank, RefSeq, and other sources.
+              Used in tools such as [<code>BLASTn</code>]({{ galaxy_base_url }}/root?tool_id=toolshed.g2.bx.psu.edu/repos/devteam/ncbi_blast_plus/ncbi_blastn_wrapper)
+          
+          - title_md: <code>NCBI nr</code> - Non-Redundant Protein Database
+            button_link: "https://www.ncbi.nlm.nih.gov/protein/"
+            button_tip: "Visit NCBI nr"
+            description_md: >
+              The NCBI protein (nr) database is a curated non-redundant protein sequence database, integrating data from multiple sources.
+              Used in tools such as [<code>BLASTp</code>]({{ galaxy_base_url }}/root?tool_id=toolshed.g2.bx.psu.edu/repos/devteam/ncbi_blast_plus/ncbi_blastp_wrapper)
+
+      - id: metagenomics
+        title: Metagenomic Databases
+        content:
+          - title_md: <code>MGnify</code> - EBI Metagenomics Database
+            button_link: "https://www.ebi.ac.uk/metagenomics/"
+            button_tip: "Visit MGnify"
+            description_md: >
+              A collection of metagenomic analyses and annotations from environmental and host-associated samples.
+              Used in tools such as [<code>Interactive MGnify Notebook</code>]({{ galaxy_base_url }}/root?tool_id=interactive_tool_mgnify_notebook)
+
 - id: workflows
   title: Workflows
-  heading_md: ''
+  heading_md: Microbiome analysis workflows to use directly, modify or take ideas from
+  content:
+    subsections:
+      - id: metabarcoding
+        title: Metabarcoding Workflows
+        content:
+          - title_md: Building an amplicon sequence variant (ASV) table from 16S data using
+              DADA2
+            description_md: Workflow covering operations related to Deposition, Variant calling,
+              Analysis, Visualisation, and DNA barcoding
+            button_link: https://dev.workflowhub.eu/workflows/1105?version=2
+            button_tip: View workflow
+            button_icon: view
+          - title_md: 'Training: 16S rRNA Analysis with Nanopore Sequencing Reads'
+            description_md: Workflow covering operations related to Sequence composition calculation,
+              Validation, Visualisation, Sequence contamination filtering, Taxonomic classification,
+              Statistical calculation, and Sequencing quality control
+            button_link: https://dev.workflowhub.eu/workflows/1064?version=1
```

Comment:
Can you build all (dev.)WorkflowHub link as the first one? Also changing the buttons

Reply:
the link you changed directed to a different workflow

Reply:
the link  imported "Marine Omics identifying biosynthetic gene clusters" workflow
but
it should import "Building an amplicon sequence variant (ASV) table from 16S data using DADA2" workflow